tests/utils.go

```diff
@@ -2284,9 +2284,95 @@ func GetDefaultVirtControllerDeployment(namespace string, config *util.KubeVirtD
 }
 
 func GetDefaultVirtHandlerDaemonSet(namespace string, config *util.KubeVirtDeploymentConfig) (*v12.DaemonSet, error) {
-	return components.NewHandlerDaemonSet(namespace, config.GetImageRegistry(), config.GetImagePrefix(), config.GetHandlerVersion(), "", "", "", config.GetLauncherVersion(), config.VirtHandlerImage, config.VirtLauncherImage, config.GetImagePullPolicy(), config.GetImagePullSecrets(), nil, config.GetVerbosity(), config.GetExtraEnv())
+	return components.NewHandlerDaemonSet(namespace, config.GetImageRegistry(), config.GetImagePrefix(), config.GetHandlerVersion(), "", "", "", config.GetLauncherVersion(), config.GetPrHelperVersion(), config.VirtHandlerImage, config.VirtLauncherImage, config.PrHelperImage, config.GetImagePullPolicy(), config.GetImagePullSecrets(), nil, config.GetVerbosity(), config.GetExtraEnv(), false)
 }
 
 func GetDefaultExportProxyDeployment(namespace string, config *util.KubeVirtDeploymentConfig) (*v12.Deployment, error) {
 	return components.NewExportProxyDeployment(namespace, config.GetImageRegistry(), config.GetImagePrefix(), config.GetExportProxyVersion(), "", "", "", config.VirtExportProxyImage, config.GetImagePullPolicy(), config.GetImagePullSecrets(), config.GetVerbosity(), config.GetExtraEnv())
 }
+
+func RenderTargetcliPod(name, disksPVC string) *k8sv1.Pod {
+	const (
+		disks        = "disks"
+		kernelConfig = "kernel-config"
+		dbus         = "dbus"
+		modules      = "modules"
+	)
+	hostPathDirectory := k8sv1.HostPathDirectory
+	targetcliContainer := renderPrivilegedContainerSpec(
+		fmt.Sprintf("%s/vm-killer:%s", flags.KubeVirtUtilityRepoPrefix, flags.KubeVirtUtilityVersionTag),
+		"targetcli", []string{"tail", "-f", "/dev/null"}, []string{})
+	targetcliContainer.VolumeMounts = []k8sv1.VolumeMount{
+		{
+			Name:      disks,
+			ReadOnly:  false,
+			MountPath: "/disks",
+		},
+		{
+			Name:      dbus,
+			ReadOnly:  false,
+			MountPath: "/var/run/dbus",
+		},
+		{
+			Name:      kernelConfig,
+			ReadOnly:  false,
+			MountPath: "/sys/kernel/config",
+		},
+		{
+			Name:      modules,
+			ReadOnly:  false,
+			MountPath: "/lib/modules",
+		},
+	}
+	return &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+			Labels: map[string]string{
+				v1.AppLabel: "test",
+			},
+		},
+		Spec: k8sv1.PodSpec{
+			RestartPolicy: k8sv1.RestartPolicyNever,
+			Containers:    []k8sv1.Container{targetcliContainer},
+			Volumes: []k8sv1.Volume{
+				// PVC where we store the backend for the SCSI disks
+				{
+					Name: disks,
+					VolumeSource: k8sv1.VolumeSource{
+						PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
+							ClaimName: disksPVC,
+							ReadOnly:  false,
+						},
+					},
+				},
+				{
+					Name: dbus,
+					VolumeSource: k8sv1.VolumeSource{
+						HostPath: &k8sv1.HostPathVolumeSource{
+							Path: "/var/run/dbus",
```

Comment:
Probably some stupid questions from a person not very familiar with targetcli and how it works :)

1. Do I understand correctly that here `targetcli` will talk to a deamon running on the **node** through the system dbus socket?
2. Why does a cli tool need `/sys/kernel/config` and `/lib/modules`?

Reply:
`targetcli` needs to load the kernel module `tcm_loop` and mount the configs under /sys/kernel/config/

Reply:
Actually, I'm not sure if it is targetcli that is doing it automatically or kubevirtci has already those configured, but those are pre-steps. I just realized I don't have in the test :/ I need to check if I need to check the environment and eventually execute them. I have created a simple [gist](https://gist.github.com/alicefr/c2e4221d7c8834a2b8746d510692d86c) to list the steps for the device creation

Reply:
ok, I just checked and `tcm_loop` isn't automatically loaded in kubevirtci, so it needs to be targetcli that does it

Reply:
> modprobe tcm_loop

Isn't the module loaded automatically when targetcli tries to use it?

> mount -t configfs configfs /sys/kernel/config/

I think for that you do not need to use HostPath. I think you can just create a directory and mount configs in the container. Or maybe use an EmptyDir instead? 


Still, why does it need dbus from the node?

Reply:
Even if it is targetcli that automatically loads the module, it needs to have access to the module to load.
I can try to add empty dir for mounting configs, thanks for the suggestion!
When I was playing with targetcli, it was requiring having access to the socket. It was at the begging, I'll double-check again if we can avoid it. Just targetcli is a generic utility for dealing with SCSI disks and it still might requiring for other operations

Reply:
I have double-checked (using the setup available[here](https://github.com/alicefr/targetcli-containers/blob/main/start-container.sh)) without the path for configs seems working, however without access to dbus, I get an error: `g-io-error-quark: Could not connect: No such file or directory (1)
`